### ISSUE TITLE
Nightly builds

### ISSUE THREAD

**Issue by cameronwhite** [flathub/com.github.PintaProject.Pinta#42]
As suggested in https://github.com/PintaProject/Pinta/issues/880, it would be great to have nightly builds available automatically in a beta branch

**Comment by darkdragon-001** [comment]
It's probably woth reading https://github.com/flathub/flathub/issues/2975#issuecomment-1086895982

**Comment by cameronwhite** [comment]
Thanks, that makes sense. So it sounds like creating periodic beta releases would be the recommended approach?

**Comment by darkdragon-001** [comment]
You can also upload artifacts in GitHub actions (Flatpak bundles) which people could download from there and install via flatpak (the corresponding action is also linked in the linked issue). They do expire after some time though and might not be so accessible for people not familiar with GitHub. Periodic beta releases are also an option. I guess if mostly depends on the intended users.

**Comment by deusnovus** [comment]
> Thanks, that makes sense. So it sounds like creating periodic beta releases would be the recommended approach?

My apologies @cameronwhite : periodic beta releases is what I meant to say in my original ticket, not automatically generated builds from the master branch!

**Comment by cameronwhite** [comment]
There is now a `3.0-beta1` tagged release, so it'd be great to try building that for the beta channel and figure out if there are any packaging issues etc with the GTK4 port of Pinta

**Comment by darkdragon-001** [comment]
As per my previous comments, if you don't plan to maintain a beta channel (for example making sure that the beta branch is not older than the stable branch in order to avoid confusing users), it might be better to just create a draft PR with the beta release and use the CI results for testing purposes.

**Comment by cameronwhite** [comment]
IMO it's reasonable to at least have the beta branch keep up with the stable branch. The goal with using the beta branch would be to have more users testing the development version of Pinta, but the CI results would expire after some time IIRC?

**Comment by darkdragon-001** [comment]
> IMO it's reasonable to at least have the beta branch keep up with the stable branch. The goal with using the beta branch would be to have more users testing the development version of Pinta, but the CI results would expire after some time IIRC?

Correct. If you want to maintain the "beta" branch, I think you should have permissions to push it. If it doesn't work, we have to open an issue in the flathub/flathub repo.

**Comment by cameronwhite** [comment]
PR #45 has set up the beta branch
